dspace-api/src/main/java/org/dspace/service/ClientInfoService.java

```diff
@@ -43,4 +43,13 @@ public interface ClientInfoService {
      */
     boolean isRequestFromTrustedProxy(String ipAddress);
 
+    /**
+     * Whether a request is from a trusted proxy or not. Only returns true if trusted proxies are specified
+     * and the ipAddress (remote address or from X-forwarded-for header) is contained in those proxies. False in all
+     * other cases
+     * @param request The request to check if it is from trusted proxy
+     * @return true if trusted, false otherwise
+     */
+    boolean isRequestFromTrustedProxy(HttpServletRequest request);
```

Comment:
This new `isRequestFromTrustedProxy` method is not used anywhere.  I believe it can be removed from here & from `ClientInfoServiceImpl`, as it's not necessary anymore.